playwright.config.ts

```diff
@@ -9,15 +9,9 @@ if (!process.env.CI) {
 export default defineConfig({
   testDir,
   outputDir: path.join(testDir, 'test-results'),
-  // We don't want this set to true as that would make tests in each file to run
-  // in parallel, which will cause conflicts with the "global state". With this
-  // set to false and workers > 1, multiple test files can run in parallel, but
-  // tests within a file are run at one at a time. We make extensive use of
-  // worker-scope fixtures and beforeAll hooks to achieve best performance.
-  fullyParallel: false,
+  fullyParallel: true,
   forbidOnly: !!process.env.CI,
   retries: process.env.CI ? 2 : 0,
-  workers: process.env.CI ? 1 : undefined,
```

Comment:
Can we have a number of max workers with `fullyParallel: true`?

Reply:
The default is half the available CPU cores. So, on GitHub it means `2` (there's 4 cores).
Can specify workers still, but better use as many as we can for speed. Can specify `--workers=N` from CLI if needed for one-off tests.